.eslintignore

```diff
@@ -1 +1,2 @@
 node_modules
+test/list-exports-tests/fixtures
```

Comment:
please please ensure all files have a trailing newline.

Reply:
Do you mind if I add an editorconfig with this configured to the repo?

```editorconfig
root = true

[*]
trim_trailing_whitespace = true
insert_final_newline = true
```

Reply:
I'de also add:
```
[*.md]
trim_trailing_whitespace = false
```

Reply:
Both are fine, thanks :-)

Reply:
actually i have no idea why we'd want that on markdown files; *everything* should trim trailing whitespace.

Reply:
Trailing whitespace is significant in markdown: more than two trailing spaces triggers a line break in markdown ([source](https://daringfireball.net/projects/markdown/syntax#p)).